src/docs/environment/index.mdx

```diff
@@ -68,34 +89,27 @@ cat /Users/`id -un`/Library/Group\ Containers/group.com.docker/settings.json | g
 # increase configured memory to something reasonable
 sed -i .bak 's/"memoryMiB":.*/"memoryMiB": 7168,/g' /Users/`id -un`/Library/Group\ Containers/group.com.docker/settings.json
 
-# check configuration 
+# check configuration
 cat /Users/`id -un`/Library/Group\ Containers/group.com.docker/settings.json | grep "memoryMiB"
 
 # start up docker with next steps
 ```
 
-On Mac, `docker` (which brew has already installed for you under `/Applications/Docker.app`) needs some manual
-intervention. You can run this command to set it up automatically for you:
-
-```shell
-open -g -a Docker.app
-```
-
-You should soon see the Docker icon in your macOS menubar. Docker will automatically run on system restarts,
-so this should be the only time you do this.
-
 You can verify that Docker is running by running `docker ps` in your terminal.
 
 ## Build Toolchain
 
-Sentry depends on [Python Wheels](https://pythonwheels.com/) (packages containing binary extension modules),
-which we distribute for the following platforms:
+Sentry depends on binary Python packages called [Python Wheels](https://pythonwheels.com/), which we distribute for the following platforms:
```

Comment:
pythonwheels.com isn't any sort of official resource, we maybe should find something on python.org? maybe the wheel PEP?